provision.py

```diff
@@ -0,0 +1,67 @@
+
+
+import sys
+import time
+import requests
+
+api_token = 'd073ebb787d57edfb03978525927cc41fac6f4bcbb0bedb65ae54fc3c6c6e482'
+api_url_base = 'https://api.digitalocean.com/v2/'
+filename = "id_rsa.pub"
+with open(filename, 'r') as f:
+    ssh_key = f.readline()
+
+# ssh_key = {'name': sys.argv[1], 'public_key': ssh_key}
+headers = {'Content-Type': 'application/json',
+           'Authorization': 'Bearer {0}'.format(api_token)}
+
+
+def create(name):
+    data = {
+            "name": name,
+            "region": "blr1",
+            "size": "s-1vcpu-1gb",
+            "image": "ubuntu-16-04-x64",
+            "ssh_keys": None,
+            "backups": False,
+            "ipv6": True,
+            "user_data": None,
+            "private_networking": None,
+            "volumes": None,
+            "tags": [
+                name
```

Comment:
it should be "name:servername" instead of "servername". You should be able to login via ssh key too.